config/karma/shared.karma.conf.js

```diff
@@ -130,7 +130,12 @@ function getConfig(config) {
     captureTimeout: 3e5,
     autoWatch: false,
     singleRun: true,
-    failOnEmptyTestSuite: false
+    failOnEmptyTestSuite: false,
+    client: {
+      jasmine: {
+        random: false
```

Comment:
Do we need to turn off random here and in `jasmine.json`?

Reply:
yeah, the `coverage:builder` tests don't use karma so they still run randomly unless you set it in `jasmine.json` too